doc/classes/AudioStreamSample.xml

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioStreamSample" inherits="AudioStream" category="Core" version="3.2">
 	<brief_description>
-		Plays audio.
+		Audio stream driver for WAV files and non-realtime dynamically generated PCM audio data.
```

Comment:
This replacement sentence sounds like a lot of complicated words that new people won't understand nor really care about. Perhaps `Stores audio data`, or something similar? 

Also, I'm not sure the brief description should be the same as the description (or just the first sentence from the description) -- while you do cite a place where the two are the same, I don't know if many of the other documentation descriptions do this. If the two were always supposed to be the same, then there would be no point to the brief description.

Reply:
I kind of agree. Most of the new info are implementation deatils, maybe move them to the bottom of the full description, but I'd keep them out of the brief description. Maybe keep it as "Plays audio." or "Plays audio, can loop." for now? Not sure.

Reply:
I agree with both of your points @LikeLakers2.

I think "Stores audio data" is more accurate than the existing comment while also being short.

As I discovered when reading the Custom AudioStream docs (mentioned below in PR thread), I think one point of confusion is due to the fact that the class *does* both store & play audio but only the storage part is directly exposed to people using the class. The playback functionality is (as far as I am aware) is only exposed to (and via) the AudioStreamPlayer.

So, for non-engine developers, "Stores audio data" seems like a correct high level summary.

The implementation detail that the class also makes it possible to play the audio seems like an aside that can be made near the end of a longer description.

Thus, a structure more like:

> Stores audio data.
>
> Stores audio data loaded from WAVE (`.wav`) format audio files which can be played with an AudioStreamPlayer.
>
> [Code example.]
>
> Advanced developers can also generate the PCM format data directly. [something something]
>
> [A note for engine developers that the "playback" functionality is also in this class but accessed via AudioStreamPlayer]

Incidentally, I think the icon of this node in the IDE is also affected by this confusion (especially when compared to the OGGVorbis node).

Reply:
I personally think that description could work, but it might benefit from a couple changes. Maybe something like this?
> AudioStreamSample stores audio data loaded from .WAV files, and can be used to play back the sound through an AudioStreamPlayer (for background music) or AudioStreamPlayer2D/AudioStreamPlayer3D (for positional audio).
> 
> This class can also be used to store PCM audio data that is generated dynamically through code.

As the class is generally made and configured through the IDE's inspector, I don't really think a code example is needed here -- also, if a developer really needs to use code to manipulate this class, I would imagine the code needed would be pretty obvious.

I'm personally unsure about having the note for engine developers be in the documentation -- perhaps that might go better in the actual .cpp/.h file defining AudioStreamSample, where developers would already be looking?

Also, if I can also make another suggestion for the brief description (this could use a tiny bit of rewording, but I believe it gets across its function better while still being easy to understand):
> Stores audio data, which is either loaded from .WAV files or generated in code.

Reply:
The brief and long descriptions should not be the same. Typically, the brief version should be simpler.